apps/component-examples/src/utils/simple-jsx.ts

```diff
@@ -0,0 +1,54 @@
+// Simple JSX renderer for HTML generation
+export interface SimpleElement {
+  type: string;
+  props: Record<string, any>;
+  children: (SimpleElement | string)[];
+}
+
+export function h(
```

Comment:
Do we think that this file is going to need much maintenance? Is there a package that can do this instead? It's good to keep react out of the examples project but maybe we can still import `h` from react/jsx?